lib/mix/test/mix/cli_test.exs

```diff
@@ -52,6 +52,15 @@ defmodule Mix.CLITest do
 
       assert contents =~ "Hello from MyProject!\n"
       refute contents =~ "This won't appear"
+
+      contents = mix ~w[my_hello], [{"MIX_QUIET", "0"}]
```

Comment:
All the sigil definitions in this file were using `~w[]` so I preferred to stick with it.
After checking, `cli_test.exs` and `test_helpers.exs` are the only files which use
`~w[]`, so we could fix this, but I think it would be better to make it as another PR,
as it is not really related to the original issue anymore.